types/models/Card.ts

```diff
@@ -0,0 +1,33 @@
+export interface Type {
+  id: number;
+  name: string;
+  suggested_labels: string | null;
+  display_icon: boolean;
+  category: string;
+  icon_url: string;
+  order: number;
+}
```

Comment:
create new file for this

`CardFieldType`

Reply:
Done